priam/src/main/java/com/netflix/priam/config/IConfiguration.java

```diff
@@ -1111,6 +1111,26 @@ default BackupsToCompress getBackupsToCompress() {
         return BackupsToCompress.ALL;
     }
 
+    /**
+     * @return true if Priam should skip deleting ingress rules for IPs not found in the token
+     *     database.
+     */
+    default boolean skipDeletingOthersIngressRules() {
+        return false;
+    }
+
+    /**
+     * @return true if Priam should skip updating ingress rules for ips found in the token database.
+     */
+    default boolean skipUpdatingOthersIngressRules() {
+        return false;
+    }
+
+    /** @return get the threshold at which point we might risk not getting our ingress rule set. */
+    default int getACLSizeWarnThreshold() {
+        return 500;
```

Comment:
Should we have a slightly lower threshold like 400 or 450 (given 500 is the known aws limit)

Reply:
I removed this altogether, began publishing warnings when we (possibly) fail to update ingress rules, and began publishing a gauge of the size of the list every time it is measured.